src/usr/local/emhttp/plugins/tailscale/include/tailscale-lock/signing.php

```
@@ -1,5 +1,14 @@
 <?php
-$tr = $tr ?? new Tailscale\Translator();
+
+namespace Tailscale;
+
+use EDACerton\PluginUtils\Translator;
+
+if ( ! defined(__NAMESPACE__ . '\PLUGIN_ROOT') || ! defined(__NAMESPACE__ . '\PLUGIN_NAME')) {
+    throw new \RuntimeException("Common file not loaded.");
+}
+
+$tr = $tr ?? new Translator(PLUGIN_ROOT);
```

PHPStan (2.1.15)

Instantiated class EDACerton\PluginUtils\Translator not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)